lib/spack/spack/cmd/doc.py

```diff
@@ -24,6 +24,8 @@
 ##############################################################################
 
 description = "run pydoc from within spack"
```

Comment:
It just runs `pydoc` (aka `help()`) within the Spack system path.  e.g., try: `spack doc spack.spec.Spec`

How about `spack pydoc`?